lib/pdksync.rb

```diff
@@ -341,7 +313,9 @@ def self.run_command(output_path, command)
   def self.pdk_update(output_path)
     # Runs the pdk update command
     Dir.chdir(output_path) unless Dir.pwd == output_path
-    _stdout, stderr, status = Open3.capture3("#{return_pdk_path} update --force")
+    template_ref = configuration.module_is_authoritive ? return_template_ref : configuration.pdk_templates_ref
+    change_module_template_url(configuration.pdk_templates_url) unless configuration.module_is_authoritive
+    _stdout, stderr, status = Open3.capture3("#{return_pdk_path} update --force --template-ref=#{template_ref}")
```

Comment:
As we are hardcoding —template-ref into this command users with custom templates will always have to specify a template_ref or they may fail due to the default being set to the PDK::version. If we omit the flag unless a custom ref is specified we could avoid this issue. It’s a bit more of a consistent workflow for both pdk template and custom template users.

Reply:
I get your workflow and I do think it makes sense, my thinking was mostly around the use of a tag in the case of template_ref, but after checking it out locally it would have the same effect as a pdk update without a flag either way. Thanks for the readme update too 👍Ping me on the PR when you're happy and I'll re-review this in view of merge

Reply:
@HelenCampbell can you give this a re-review?